src/DotNetty.Common/Utilities/ArrayExtensions.cs

```diff
@@ -72,5 +72,44 @@ public static void Fill<T>(this T[] array, int offset, int count, T value)
                 array[i] = value;
             }
         }
+
+        /// <summary>
+        ///     Merge the byte arrays into one byte array.
+        /// </summary>
+        public static byte[] CombineBytes(this byte[][] arrays)
+        {
+            int newlength = 0;
+            arrays.ForEach(delegate (byte[] array) { newlength += array.Length; });
```

Comment:
TBH, looking at actual three lines of ForEach body, pls just use `foreach (var item in array)` in place and don't add any extension methods for such a trivial thing.